### ISSUE TITLE
Remove "Logout" button

### ISSUE THREAD

**Issue by kenancasey** [rgt731/Chapp#10]

**Comment by rgt731** [comment]
How would they log out though?

**Comment by kenancasey** [comment]
I agree it should be somewhere. Is there another place to hide it? Like a profile view?

**Comment by rgt731** [comment]
there isnt a profile view currently. I could put it in home.

**Comment by kenancasey** [comment]
What if you remove the ugly nav icon, add a large left-aligned title "FHU Chapel", and add the profile button in the upper right?
